styles/components/_notice.scss

```
@@ -0,0 +1,24 @@
+.notice {
+  background-color: $focus;
+  width: 90%;
```

```suggestion
  width: 100%;
```
Might look nicer full width, IMO.

---

Moved to bottom of the page and kept at the original widths.